src/packages/database/test/model.test.js

```diff
@@ -382,6 +382,35 @@ describe('module "database/model"', () => {
         // $FlowIgnore
         expect(await result.user).to.have.property('id', user.getPrimaryKey());
       });
+
+      it('constructs and persists a `Model` instance with an integer id specified', async () => {
+        const user = new User({ id: 1 });
+        const body = 'Contents of "Test Post"...';
+        const title = 'Test Post';
+        const id = 999;
+
+        result = await Subject.create({
+          id,
+          user,
+          body,
+          title,
+          isPublic: true
+        });
+
+        expect(result).to.be.an.instanceof(Subject);
```

Comment:
I feel like this assertion and `expect(result).to.have.property('id', id);` from L403 are all that is required to ensure that you can manually insert ids. Maybe w could remove all of the other expect calls and the unnecessary constants for this test case.